apps/website/src/lib/components/image/image.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import type { HTMLImgAttributes } from 'svelte/elements';
+  import { onMount } from 'svelte';
+
+  // eslint-disable-next-line
```

Comment:
I _should_ be more specific here about which thing I am disabling, but I don't really care.